tests/deployment_test.go

```diff
@@ -309,6 +309,24 @@ func (s *DeploymentSuite) verifyDeploymentListInfo(expectedDeploymentListInfo *d
 	return true
 }
 
+func (s *DeploymentSuite) listDeploymentsAllPages(ctx context.Context, request *workflowservice.ListDeploymentsRequest) ([]*deploymentpb.DeploymentListInfo, error) {
```

Comment:
```suggestion
func (s *DeploymentSuite) listDeploymentsAll(ctx context.Context, request *workflowservice.ListDeploymentsRequest) ([]*deploymentpb.DeploymentListInfo, error) {
```
akin to `io.ReadAll`?